multicluster/controllers/multicluster/member/gateway_controller.go

```diff
@@ -176,9 +191,19 @@ func (r *GatewayReconciler) createResourceExport(ctx context.Context, req ctrl.R
 		PodCIDRs:    r.podCIDRs,
 		GatewayInfos: []mcsv1alpha1.GatewayInfo{
 			{
-				GatewayIP: gatewayIP,
+				GatewayIP: gateway.GatewayIP,
 			},
 		},
+		WireGuard: mcsv1alpha1.WireGuardConfig{
+			PublicKey: gateway.Annotations[antreaTypes.MulticlusterWireGuardPublicAnnotationKey],
```

Comment:
Why key and port are annotations, but not attributes of Gateway?

Reply:
Hi Jianjun, thanks for the advice. I removed the annotations and add a new field in Gateway. In this way, we can avoid some convert from string to int.

Now, the Gateway looks like this:

```yaml
- apiVersion: multicluster.crd.antrea.io/v1alpha1
  gatewayIP: 172.18.0.5
  internalIP: 172.18.0.5
  kind: Gateway
  metadata:
    creationTimestamp: "2023-02-21T18:01:13Z"
    generation: 2
    name: east-control-plane
    namespace: kube-system
    resourceVersion: "1496"
    uid: a6dc4684-dd0a-45c1-a971-f87df8306114
  wireGuard:
    port: 51821
    publicKey: ygRS38u7gbbvAlVx4Pf9FAhI9Z2KHuvgfKQe1g2K1Fg=
```

Reply:
Sounds great to me.